recipes/openmesh/sitedir.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/src/Python/CMakeLists.txt b/src/Python/CMakeLists.txt
+index e622ab8..22fb44a 100644
+--- src/Python/CMakeLists.txt
++++ src/Python/CMakeLists.txt
+@@ -89,8 +89,12 @@ IF(OPENMESH_BUILD_PYTHON_BINDINGS)
+ 
+ 						FILE(GLOB SOURCES *.cc *hh)
+ 						ADD_LIBRARY(openmesh SHARED ${SOURCES})
+-
+-						install(TARGETS openmesh DESTINATION ${ACG_PROJECT_LIBDIR}/python )
++                        if (WIN32)
```

Comment:
Sure, and when it is fixed upstream then we also benefit. But less patches are better as its less things to manage. And in this case the patch doesn't actually fix anything for building, it just installs the file to a different place, which we can already achieve by moving post-build.